src/main/java/net/snowflake/client/jdbc/RestRequest.java

```diff
@@ -345,7 +376,7 @@ public static CloseableHttpResponse execute(
 
         // check canceling flag
         if (canceling != null && canceling.get()) {
-          logger.debug("Stop retrying since canceling is requested", false);
+          logger.debug(requestIdStr + "Stop retrying since canceling is requested", false);
```

Comment:
why don't we have parameters here as `{}`? is the `false` parameter missing in the log?

Reply:
`false` parameter is a flag used to mask the secrets, but as we do not have any secrets here, I have changed the message to a proper format.